StackAsLinkedList.java

```diff
@@ -0,0 +1,105 @@
+// Time Complexity : O(n)
+// Space Complexity : O(n)
+// Did this code successfully run on Leetcode : N/A, work with test cases
+// Any problem you faced while coding this : N/A
+

```

Comment:
Good Work! Try to implement this using O(1) time and space complexity preserving the Stack rules.